backend/pom.xml

```diff
@@ -48,9 +48,16 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>${log4j.version}</version>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
Where is the slf4j stuff needed?

Reply:
Slf4j is still used by some parts of the codebase, also by the generated Thrift code and in transitive dependencies. This import makes sure that all log statements written through the slf4j API are routed to log4j.